### PR TITLE
[ARCTIC-860] Improve  memory usage when plan table

### PR DESCRIPTION
**Pull request by shidayang**
## Why are the changes needed?
fix: #860 

## Brief change log
BaseUnkeyedTable get TableScan without includeColumnStats

## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (yes / **no**)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / **not documented**)

### CODE REVIEW

File: core/src/main/java/com/netease/arctic/data/DefaultKeyedFile.java [hunk 1]
```diff
@@ -78,18 +78,12 @@ public Long transactionId() {
 
   @Override
   public ChangedLsn minLsn() {
```
Comment (zhoujinsong) [inline core/src/main/java/com/netease/arctic/data/DefaultKeyedFile.java:80]: Maybe we can remove this method from the interface if it is not needed for now.

**Review by zhoujinsong — COMMENTED** [review]
@shidayang I left some comments. BTW, I am wondering how much memory can we sav if we skip the file stat when reading the table.

**Comment by zhoujinsong** [comment]
@shidayang Can you rename the PR title to match community requirements？

File: core/src/main/java/com/netease/arctic/table/BaseUnkeyedTable.java [hunk 2]
```diff
@@ -101,7 +101,7 @@ public void refresh() {
 
   @Override
   public TableScan newScan() {
-    return icebergTable.newScan().includeColumnStats();
+    return icebergTable.newScan();
```
Comment (wangtaohz) [inline core/src/main/java/com/netease/arctic/table/BaseUnkeyedTable.java:104]: Does Iceberg's TableScan not `includeColumnStats` as default?

Comment (shidayang) [reply core/src/main/java/com/netease/arctic/table/BaseUnkeyedTable.java:104]: yes

**Comment by shidayang** [comment]
@zhoujinsong From 190_989_776 to 16_682_384 in my case with about 8000+ files

**Comment by zhoujinsong** [comment]
> @zhoujinsong From 190_989_776 to 16_682_384 in my case with about 8000+ files

Cool! That really is a big improvement!

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**